v23.1/alter-backup.md

```diff
@@ -71,6 +71,28 @@ ALTER BACKUP '2022/03/23-213101.37' IN 's3://{BUCKET NAME}?AWS_ACCESS_KEY_ID={KE
 
 To list backup directories at a collection's URI, see [`SHOW BACKUP`](show-backup.html).
 
+### Add an Azure KMS key to an encrypted backup
+
+{% include_cached new-in.html version="v23.1" %} To add a new KMS key to the most recent backup:
```

Comment:
```suggestion
{% include_cached new-in.html version="v23.1" %}

To add a new KMS key to the most recent backup:
```

Current text made it look like the only new functionality is that first sentence: adding to the 'most recent backup'.

So I'm hoping it works to give the first sentence a new paragraph, i.e. make it a bit clearer that 'new in' applies to the whole section.

However, I'm not sure if our script that we use to set up the next release and strip these 'new in' mentions knows to look for instances of the include followed by newline characters. If that's the only downside, and you choose to implement this change, can you ping me to confirm you're doing so, so I can look into this?

Reply:
@mikeCRL I have implemented this instance of version tag on its own line